### PR TITLE
Lifestyle/staying healthy topic bundle

### PR DESCRIPTION
**Pull request by Blizzardsev**
This will introduce the following:

- Five new topics; `talk_using_computers_healthily`, `talk_staying_active`, `talk_relieving_stress`, `talk_careful_spending`, `talk_eating_well`. All of these are locked to `HAPPY`+ affinity.
- Supporting code for conditional checks for new topics.

### CODE REVIEW

File: game/script-ch30.rpy [hunk 1]
```diff
@@ -26,6 +26,14 @@ label ch30_visual_setup:
     #FALL THROUGH
 
 label ch30_init:
+
+    # Add to the total visits counter
+    $ persistent.jn_total_visit_count += 1
+
+    # Set the last visited date, if not already on record
+    if not persistent.jn_first_visited_date:
+        $ persistent.jn_first_visited_date = datetime.datetime.now()
```
Comment (spaghetsie) [inline game/script-ch30.rpy:35]: Might be better to do in `splashscreen` so we avoid checking for it on every cycle

File: game/script-topics.rpy [hunk 2]
```diff
@@ -591,6 +591,231 @@ label talk_service_animals:
...
+label talk_using_computers_healthily:
+    n "Huh."
+    n "Hey,{w=0.1} [player].{w=0.2} I just thought of something."
+    n "You gotta be at your computer to talk to me,{w=0.1} right?"
+    n "And you've been here a while already..."
+    n "Alright,{w=0.1} that's it!{w=0.2} I've decided."
+    n "I'm gonna give you a little lesson on using your computer the right way!"
+    n "Number one:{w=0.2} posture!"
+    n "Sit up straight,{w=0.1} and back against the chair,{w=0.1} [player].{w=0.2} I mean it!"
+    n "You don't want back problems,{w=0.1} do you?"
```
Comment (spaghetsie) [inline game/script-topics.rpy:620]: too late nat :p

File: game/script-ch30.rpy (hunk 1 above)
Comment (Blizzardsev) [reply game/script-ch30.rpy:35]: Can do, will get that added in next commit

File: game/script-topics.rpy (hunk 2 above)
Comment (Blizzardsev) [reply game/script-topics.rpy:620]: jahy_didmybest

**Review by spaghetsie — DISMISSED** [review]
I haven't run it myself yet but it seems bug-free from looking at it
~famous last words~

**Review by spaghetsie — APPROVED** [review]
*bigpats*

**Pull request merged**